MTA-7229 [DOC] Configuring repositories documentation is misleading (#384)
migtools/mta-documentation: docs/topics/mta-ui/proc_configuring-subversion-repos.adoc

:_mod-docs-content-type: PROCEDURE
[id="configuring-subversion-repos_{context}"]
= Enable insecure access to Subversion repositories

[role="_abstract"]

> Administrators can configure {ProductFullName} to accept self-signed or internal certificates to allow an insecure connection to access the *Subversion* repository internal to your organization.

Reviewer: ```suggestion
As an Administrator, configure {ProductFullName} to accept self-signed or internal certificates to allow an insecure connection to your internal *Subversion* repository.
```
